tutorindigo/templates/indigo/lms/templates/courseware/course_about.html

```diff
@@ -71,7 +71,7 @@
         %else:
         <div class="media">
           <div class="hero">
-            <img src="${course_image_urls['large']}" alt="" />
+            <img src="${course_image_urls['large']}" onerror="this.src='/theming/asset/images/no_course_image.png';" alt="" />
```

Comment:
I think we should indicate in a comment that this is an Indigo-specific part. Otherwise it's going to be difficult to figure out in the future when we upgrade. (same for change below)